modules/uhttp/filters_test.go

```diff
@@ -42,14 +42,58 @@ import (
 	"github.com/uber/jaeger-client-go/config"
 )
 
-func TestFilterChain(t *testing.T) {
+func TestDefaultFiltersWithNopHost(t *testing.T) {
+	// setup
 	host := service.NopHost()
+	stats.SetupHTTPMetrics(host.Metrics())
+
+	t.Run("run parallel", func(t *testing.T) {
```

Comment:
we cannot set t.Parallel at line 46, since `stats.SetupHTTPMetrics(host.Metrics())` set global, no test should run parallel when they are going to touch this value. so `TestDefaultFiltersWithNopHost` `TestDefaultFiltersWithNopHostAuthFailure` and `TestDefaultFiltersWithNopHostConfigured` have to run one after another